Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM openshift/origin-release:golang-1.10 as builder
+WORKDIR /go/src/github.com/operator-framework/operator-marketplace
+COPY . .
+RUN make build
```

Comment:
Is multi-stage builds being done in other operator repos?

Reply:
I'm going off this example, which uses the multistage build:

https://github.com/openshift/console-operator/blob/master/Dockerfile